subiquity/client/controllers/filesystem.py

```diff
@@ -287,21 +291,18 @@ async def _answers_action(self, action):
         else:
             raise Exception("could not process action {}".format(action))
 
-    async def _guided_choice(self, choice: Optional[GuidedChoiceV2]):
-        if self.core_boot_capability is not None:
-            self.app.next_screen(self.endpoint.guided.POST(choice))
-            return
+    async def _guided_choice(self, choice: GuidedChoiceV2):
         # FIXME It would seem natural here to pass the wait=true flag to the
         # below HTTP calls, especially because we wrap the coroutine in
         # wait_with_progress.
         # Having said that, making the server return a cached result seems like
         # the least risky option to address https://launchpad.net/bugs/1993257
         # before the kinetic release. This is also similar to what we did for
         # https://launchpad.net/bugs/1962205
```

Comment:
@ogayot @dbungert i have a feeling I've removed the code this comment was about. Do we need to add a use_cached_result parameter to guided.POST?

Reply:
since https://github.com/canonical/subiquity/pull/1234 (addressing 1962205), guided.POST (the v1) always returns a cached response. I think the comment was specifically about the GET, for which we don't always have a cached response.

Reply:
Thanks for checking for me. I'll delete the comment